hello-world-attest-fn-call/README.md

```diff
@@ -0,0 +1,267 @@
+# Hello World
+
+This example shows you how to use the Blocky Attestation Service (Blocky AS) to
+attest a simple function calls.
+
+You'll learn how to:
+
+- Create a function that returns a `"Hello, World!"` message
+- Create a function that writes a message to a log and returns a
+  user-defined error
+- Invoke functions in the Blocky AS using its `bky-as` CLI
+- Extract function output from the Blocky AS attestation
+
+## Setup
+
+- Install the Blocky AS CLI by following the
+  [setup instructions](https://blocky-docs.redocly.app/attestation-service/setup)
```

Comment:
should this be a relative link?  Seems like use absolute links within the project could lead to some broken link issues.

Reply:
I'm not sure I follow. We are in the examples repo, not the docs here, so I don't think we have the option of a relative link